strongbox-web-core/src/main/java/org/carlspring/strongbox/rest/TrashRestlet.java

```diff
@@ -1,25 +1,28 @@
 package org.carlspring.strongbox.rest;
 
-import io.swagger.annotations.*;
 import org.carlspring.strongbox.services.ArtifactManagementService;
 import org.carlspring.strongbox.storage.resolvers.ArtifactStorageException;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.stereotype.Component;
 
 import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import java.io.File;
 import java.io.IOException;
 
+import io.swagger.annotations.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Component;
+
 /**
  * @author Martin Todorov
  */
 @Component
 @Path("/trash")
 @Api(value = "/trash")
+@PreAuthorize("permitAll")
```

Comment:
@AlexOreshkevich ,
I think this needs to be furter fine-grained. Not everyone will be allowed to clear the trash. There will have to be a privilege for this as well.